client_dashboard/common/src/commonMain/kotlin/org/thechance/common/presentation/restaurant/RestaurantFilterDropDownMenu.kt

```diff
@@ -0,0 +1,123 @@
+package org.thechance.common.presentation.restaurant
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.isSystemInDarkTheme
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import com.beepbeep.designSystem.ui.composable.BpOutlinedButton
+import com.beepbeep.designSystem.ui.composable.BpTextButton
+import com.beepbeep.designSystem.ui.theme.Theme
+import org.thechance.common.components.EditablePriceBar
+import org.thechance.common.presentation.composables.BpDropdownMenu
+import org.thechance.common.presentation.composables.EditableRatingBar
+
+@Composable
+fun RestaurantFilterDropDownMenu(
+    expanded: Boolean,
+    rating: Double,
+    priceLevel: Int,
+    onClickRating: (Double) -> Unit,
+    onClickPrice: (Int) -> Unit,
+    onDismissRequest: () -> Unit
```

Comment:
rearrange so functions are at the beginning

Reply:
done